tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
     "compilerOptions": {
-        "target": "es2015",
-        "module": "es2015",
+        "target": "es2020",
+        "module": "es2020",
         "moduleResolution": "node",
         "esModuleInterop": true,
-        "lib": ["es2015", "es2016", "es2017", "es2018", "DOM"],
+        "lib": ["es2015", "es2016", "es2017", "es2018", "es2019", "es2020", "DOM"],
         "declaration": false,
         // Enhance Strictness
         "strict": true,
```

Comment:
Also noticed this is a more opinionated compiler options config overall than Locker. I wonder if Locker could benefit from more of this.

Reply:
+1 Even though locker and near-membrane are separate repos, we should strive to keep them the same in terms of dependencies, configs, etc.